src/IO/WriteBufferFromHTTP.cpp

```diff
@@ -10,6 +10,7 @@ WriteBufferFromHTTP::WriteBufferFromHTTP(
     const Poco::URI & uri,
     const std::string & method,
     const std::string & content_type,
+    const CompressionMethod compression_method,
```

Comment:
Better to pass `const std::string &` content_encoding. `HTTPBuffer` should not be aware of Compression.

Reply:
Updated.